test/utils/image/manifest.go

```diff
@@ -69,6 +70,7 @@ func initReg() RegistryList {
 		DockerLibraryRegistry:   "docker.io/library",
 		DockerGluster:           "docker.io/gluster",
 		E2eRegistry:             "gcr.io/kubernetes-e2e-test-images",
+		PromoterE2eRegistry:     "us.gcr.io/k8s-artifacts-prod/e2e-test-images",
```

Comment:
After the [domain flip](https://groups.google.com/d/msg/kubernetes-sig-release/ew-k9PEBckQ/T7dFepHdCAAJ), this should instead be k8s.gcr.io/k8s-artifacts-prod/e2e-test-images.

Reply:
Done

Reply:
Done

Reply:
Oh, I think that's not going to work. The flip doesn't happen until April 1!

Reply:
Oups, I shouldn't have changed that then. Changing it back and leaving a comment.